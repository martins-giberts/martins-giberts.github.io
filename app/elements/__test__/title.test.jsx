/* globals describe, it, expect */

import React from 'react'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'

import Component from '../title/index'

describe('elements/Title test', () => {
  it('should render H1 without throwing an error', () => {
    const wrapper = shallow(<Component Kind={'h1'}>Heading</Component>)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

  it('should have prop className title h2', () => {
    const wrapper = shallow(<Component Kind={'h2'}>Heading</Component>)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
