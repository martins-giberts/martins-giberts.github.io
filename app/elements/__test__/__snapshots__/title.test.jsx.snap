// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`elements/Title test should have prop className title h2 1`] = `
<h2
  className="title h2"
>
  Heading
</h2>
`;

exports[`elements/Title test should render H1 without throwing an error 1`] = `
<h1
  className="title h1"
>
  Heading
</h1>
`;
